### PR TITLE
Config Params.

### PR DESCRIPTION
**Pull request by fengttt**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

**Comment by fengttt** [comment]
I dont intent to check in this.    The purpose is to discuss what we can reduce from the params.   So here is my list, commented out means I don't want to document/release this to public.

I am asking for help, to see if we can cut more.

File: etc/launch-tae-CN-tae-DN/cn.toml [hunk 1]
```diff
@@ -27,23 +25,23 @@ data-dir = "mo-data/s3"
...
+# disk-evict-target = 0.8
 
 [[fileservice]]
 name = "ETL"
 backend = "DISK-ETL"
 
-[observability]
-disableTrace = true
-disableMetric = true
+# [observability]
```
Comment (daviszhen) [inline etc/launch-tae-CN-tae-DN/cn.toml:35]: 这地方需要谢泽雄确认下

**Review by daviszhen — APPROVED** [review]

File: etc/launch-tae-CN-tae-DN/cn.toml (hunk 1 above)
Comment (xzxiong) [reply etc/launch-tae-CN-tae-DN/cn.toml:35]: 没问题的。
在standalone启动时，要全部cn/dn/log 都配置了才能生效。所以此处实际效果等同与  false

**Review by xzxiong — APPROVED** [review]

File: etc/launch-tae-CN-tae-DN/cn.toml [hunk 2]
```diff
@@ -3,8 +3,6 @@ data-dir = "./mo-data"
 
 [log]
 level = "info"
-format = "console"
```
Comment (xzxiong) [inline etc/launch-tae-CN-tae-DN/cn.toml:6]: PS: `format` default  val = "json"

Comment (fengttt) [reply etc/launch-tae-CN-tae-DN/cn.toml:6]: OK， this seems to be a useful thing to expose?   but if format is console, not json, will this affect our system table?

**Comment by fengttt** [comment]
Discussed, and moving forward by teams.

File: etc/launch-tae-CN-tae-DN/cn.toml (hunk 2 above)
Comment (xzxiong) [reply etc/launch-tae-CN-tae-DN/cn.toml:6]: It will NOT.

**Pull request closed without merging**